frantisek_duda/Dockerfile

```diff
@@ -0,0 +1,45 @@
+#Stage 1 Build
+FROM golang:alpine as builder
+RUN apk update && apk add --no-cache  \ 
+iputils-ping \ 
+sudo \
+net-tools \
+tcpdump # version 4.99.4-3ubuntu1 \
+iproute2 \
+grep \
+bash \
+curl
+ENV PATH="/usr/sbin:${PATH}"
+WORKDIR /home/duda_fr/beeit_devops_podzim_2024/frantisek_duda
+
+COPY script.sh /home/duda_fr/beeit_devops_podzim_2024/frantisek_duda/
+RUN chmod +x /home/duda_fr/beeit_devops_podzim_2024/frantisek_duda/script.sh
+RUN go build -o frantisek_duda
```

Comment:
Image nepujde buildnout, protoze to spadne na go build a ten nepujde udelat, protoze uvnitr v dockeru bude chybet `go` script (`main.go`, `go.mod`...).
Chces zkompilovat program (tim prikazem `go build`, ale nemas ten kod, ktery chces kompilovat)